File: 0001 Code Chef/0023 Chef Plays Ludo/implementation/0003 c#/Ludo.cs
using System;

class Ludo{
    static void Main(string[] args){
        string stringInput;
        stringInput = Console.ReadLine();
        int test = Convert.ToInt32(stringInput);
        while(test > 0){
            stringInput = Console.ReadLine();
            int x = Convert.ToInt32(stringInput);
            if(x == 6){
                Console.WriteLine("YES");
            }else{
                Console.WriteLine("NO");
            }
            test = test - 1;
        }
    }
}

File: 0001 Code Chef/0023 Chef Plays Ludo/implementation/0004 java/Ludo.java
import java.util.Scanner;

public class Ludo{
    public static void main(String[] args){
        String stringInput;
        Scanner read = new Scanner(System.in);
        stringInput = read.nextLine();
        int test = Integer.parseInt(stringInput);
        while(test > 0){
            stringInput = read.nextLine();
            int x = Integer.parseInt(stringInput);
            if(x == 6){
                System.out.println("YES");
            }else{
                System.out.println("NO");
            }

            test = test - 1;
        }
    }
}